modules/s3-remote-state/README.md

```diff
@@ -37,43 +47,80 @@ output "bucket_name" {
 }
 ```
 
-Create a `terraform.tfvars`, for example:
+Create a `terraform.tfvars`, for example (Note that the principals are a list of arns, so even if you only use one, it has to be inside a list):
 
 ```hcl
-principals="example needed here"
-bucket_name="foobar-remote-state"
+principals=["<arn of principal>"]
+bucket_name="fpco-tf-remote-state-test"
 ```
 
-Review the changes Terraform will make with `tf plan`, and apply those changes
-with `tf apply`.
+Initially the project with `terraform init.` Review the changes Terraform will make with `tf plan --out=tf.out`, and apply those changes with `tf apply tf.out`.
+
+### Initialize the Remote State S3 Backend
 
-Configure remote state to use that bucket with:
+We want to use the newly created S3 bucket to store the Terraform state for this module. There are several steps that need to be taken. Make sure to backup your existing `terraform.tfstate` file before taking these steps:
 
+1. First we have to [initialize the backend][1] and copy over the current state of the backend bucket you just created. Uncomment the following, so that the code in the `main.tf` file looks similar to this:
```

Comment:
I'm not sure why we should recommend using this partial config method when the commented out keys a) have all that is needed, and b) end up with a simpler/clearer/more explicit result, or am I overlooking something that makes the partial config work better?

Reply:
The partial config method is 1/ recommended in TF doco and 2/ is only for the initial set up of the s3 project itself. It allows you to switch the s3 local project over to remote safely. As noted in point 3, when other people create projects, they should use the uncommented version so they will start with remote from the get go. I think the wording there is confusing and perhaps I should make that clearer and more explicit.